b02cf9e61b861d10d806dc23b24bcb3f/update/sys_script_b64906e21bc61d10d806dc23b24bcbd9.xml

```diff
@@ -31,14 +31,20 @@
 	
 	var parser = new GlideRecord('x_snc_slackerbot_parser');
 	parser.addQuery('active', 'true');
+	//Loop in decreasing order
+	parser.orderByDesc('order');
```

Comment:
This is opposite of how order behavior usually works in ServiceNow. We really should do the opposite